saleor/static/dashboard-next/productTypes/index.tsx

```diff
@@ -13,14 +13,10 @@ export const productTypeListUrl = "/productTypes";
 const ProductTypeList: React.StatelessComponent<RouteComponentProps<{}>> = ({
   location
 }) => {
-  const queryString = parseQs(location.search.substr(1));
+  const qs = parseQs(location.search.substr(1));
   const params = {
```

Comment:
How about:

```
const { after, before } = qs;

...

return <ProductTypeListComponent params={{ after, before }} />;
```

And if we're here, why is there `params` instead of explicitly accepting `after` and `before` as props?